dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,11 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.master.master-service/master-service": {master-routes:       "/puppet"
 
+                                                                 # The following controls 404 errors and should not
+                                                                 # be modified
+                                                                 invalid-in-puppet-4: "/" }
```

Comment:
Yes, this question has been discussed at great length in many forums, and we're not going to try to restrict the message to only URLs that would have been valid in old agents.